www/templates/hpc-new-user

```diff
@@ -0,0 +1,10 @@
+Hi {name},
+
+I've added you to the HPC user group, so you can now SSH into hpcctl and start running jobs.
```

Comment:
It occurs to me that hpcctl is a really unwieldy name for the single login node, we may want to change the CNAME to hpc.ocf?

Reply:
At the very least, I would suggest writing an FQDN here.